src/midi/midi_fluidsynth.cpp

```diff
@@ -101,6 +101,63 @@ std::tuple<std::string, int> parse_sf_pref(const std::string &line,
 	return {filename, percent};
 }
 
+#if defined(WIN32)
+
+static std::vector<std::string> get_data_dirs()
+{
+	return {
+	        CROSS_GetPlatformConfigDir() + "soundfonts\\",
+	        "C:\\soundfonts\\",
+	};
+}
+
+#elif defined(MACOSX)
+
+static std::vector<std::string> get_data_dirs()
+{
+	return {
+	        CROSS_GetPlatformConfigDir() + "soundfonts/",
+	        CROSS_ResolveHome("~/Library/Audio/Sounds/Banks/"),
+	        // TODO: check /usr/local/share/soundfonts
+	        // TODO: check /usr/share/soundfonts
+	};
```

Comment:
I wonder if we can add a lookup for `./sounfonts/`, essentially in dosbox's CWD/PWD.

This would allow integrators to package soundfonts along-side a set of N games, without needing external tools to copy/move sf2's into the user's conf directory or alternatively figure out the absolute path to their local `soundfonts` directory and update the conf files with those absolute paths.

Reply:
It works already ;) if soundfont file is in CWD, then it will be detected before moving on to looking in other directories.

In your example, assuming `dos` is DOSBox current working directory, user can write `soundfont = soundfonts/file.sf2` and that's it :)

Reply:
Brilliant! :-)